apertium-kaa.kaa.lexc

```diff
@@ -5831,7 +5829,7 @@ keshe:keshe ADV-WITH-KI ; ! "yesterday > yesterday's"
 kesh:kesh ADV-WITH-KI ; ! "evening/late"
 kúndiz:kúndiz ADV-WITH-KI ; ! "during the day"
 házir:házir ADV-WITH-KI ; ! "now > the one now"
-soń:soń ADV-WITH-KI ; ! ""
+soń:soń ADV; ! ""
```

Comment:
Why did you make this change?  Are you saying that there is no form like "sońǵı"?